tests/test_features.py

```diff
@@ -10,14 +15,6 @@
 except KeyError:
     pass
 
-import matplotlib
```

Comment:
Does the entire test suite still run if you cut this bit out?  I remember having trouble with the image comparison tests due to import-order/test discovery order messing up the mpl configuration.